### PR TITLE
feat: Add show service service-using

### PR DESCRIPTION
**Pull request by bennnnnnnn**
Add show service service-using command for Alcatel SROS

### CODE REVIEW

File: ntc_templates/templates/alcatel_sros_show_service_service-using.textfsm [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Value CUSTOMER_ID (\d+)
+Value SERVICE_NAME (\S+)
+
+Start
+  ^ServiceId\s+Type\s+Adm\s+Opr\s+CustomerId\s+Service\s+Name -> Services
+
+Services
+  ^\s*${SERVICE_ID}\s+${TYPE}\s+${ADMIN_STATUS}\s+${OPERATIONAL_STATUS}\s+${CUSTOMER_ID}\s+${SERVICE_NAME}\s*$$ -> Record
+  ^-+\s*$$
+  ^Matching Services : \d+\s*$$ -> End
```
Comment (mjbear) [inline ntc_templates/templates/alcatel_sros_show_service_service-using.textfsm]: If you like this suggestion, click the Commit suggestion button to accept my edit. Thank you!

```suggestion
  ^Matching Services\s+:\s+\d+\s*$$ -> End
```

File: ntc_templates/templates/alcatel_sros_show_service_service-using.textfsm [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Header
+  ^=+\s*$$
+  ^ServiceId\s+Type\s+Adm\s+Opr\s+CustomerId\s+Service\s+Name -> Services
+  ^. -> Error
+
+Services
+  ^\s*${SERVICE_ID}\s+${TYPE}\s+${ADMIN_STATUS}\s+${OPERATIONAL_STATUS}\s+${CUSTOMER_ID}\s+${SERVICE_NAME}\s*$$ -> Record
+  ^-+\s*$$
+  ^Matching Services : \d+\s*$$
```
Comment (mjbear) [inline ntc_templates/templates/alcatel_sros_show_service_service-using.textfsm:21]:
```suggestion
  ^Matching\s+Services\s+:\s+\d+\s*$$ -> End
```

File: ntc_templates/templates/alcatel_sros_show_service_service-using.textfsm [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  ^-+\s*$$
+  ^=+\s*$$ -> End
+  ^. -> Error
```
Comment (mjbear) [inline ntc_templates/templates/alcatel_sros_show_service_service-using.textfsm:24]: Simpler.
Less duplication. More DRY.

If you like this suggestion, click the Commit suggestion button to accept my edit. Thank you!

```suggestion
Start
  ^=+\s*$$
  ^Services.*$$
  ^ServiceId\s+Type\s+Adm\s+Opr\s+CustomerId\s+Service\s+Name
  ^\s*${SERVICE_ID}\s+${TYPE}\s+${ADMIN_STATUS}\s+${OPERATIONAL_STATUS}\s+${CUSTOMER_ID}\s+${SERVICE_NAME}\s*$$ -> Record
  ^Matching\s+Services\s+:\s+\d+\s*$$
  ^-+\s*$$
  ^. -> Error
```

File: ntc_templates/templates/alcatel_sros_show_service_service-using.textfsm [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+Value SERVICE_NAME (\S+)
+
+Start
+  ^=+\s*$$
+  ^Services.*$$
+  ^ServiceId\s+Type\s+Adm\s+Opr\s+CustomerId\s+Service\s+Name
+  ^\s*${SERVICE_ID}\s+${TYPE}\s+${ADMIN_STATUS}\s+${OPERATIONAL_STATUS}\s+${CUSTOMER_ID}\s+${SERVICE_NAME}\s*$$ -> Record
+  ^Matching\s+Services\s+:\s+\d+\s*$$
+  ^-+\s*$$
+  ^. -> Error
```
Comment (jmcgill298) [inline ntc_templates/templates/alcatel_sros_show_service_service-using.textfsm:15]:
```suggestion
  ^. -> Error

```

**Review by jmcgill298 — APPROVED** [review]

**Pull request merged**